packages/block-editor/src/components/block-list/block.js

```diff
@@ -168,19 +171,24 @@ function BlockListBlock( {
 	// (InspectorControls, etc.) which are inside `BlockEdit` but not
 	// `BlockHTML`, even in HTML mode.
 	let blockEdit = (
-		<BlockEdit
-			name={ name }
-			isSelected={ isSelected }
-			attributes={ attributes }
-			setAttributes={ setAttributes }
-			insertBlocksAfter={ isLocked ? undefined : onInsertBlocksAfter }
-			onReplace={ isLocked ? undefined : onReplace }
-			onRemove={ isLocked ? undefined : onRemove }
-			mergeBlocks={ isLocked ? undefined : onMerge }
-			clientId={ clientId }
-			isSelectionEnabled={ isSelectionEnabled }
-			toggleSelection={ toggleSelection }
-		/>
+		<div
+			onMouseEnter={ () => addHoveredBlock( clientId ) }
+			onMouseLeave={ () => removeHoveredBlock( clientId ) }
+		>
```

Comment:
We will probably want a better place to add these than tossing in a new div wrapper.  These seem to get overridden if they are placed in the BlockEdit component below.

Reply:
if there are existing handlers for mouse enter/leave, should we tie into those possibly?

Reply:
definitely!  I was getting a little lost trying to find those and put it here for now.  😆